sdk/communication/communication-sms/test/smsClient.mocked.spec.ts

```diff
@@ -33,7 +35,7 @@ describe("[mocked] SmsClient", async () => {
         parsedBody: {
           value: [
             {
-              to: "+18332321226",
+              to: "+15558985487",
```

Comment:
Why the change here?

Reply:
putting in a 555 just to be sure it's not exposing a real number. I guess technically since this is mocked i can grab it from the env file

Reply:
env file is for node, recorder replaces it for live tests, so i'm just using the number we have defined in the recorder